langworld_db_data/removers/listed_value_remover.py

```diff
@@ -0,0 +1,134 @@
+from langworld_db_data.constants.literals import ID_SEPARATOR
+from langworld_db_data.filetools.csv_xls import read_dicts_from_csv, write_csv
+from langworld_db_data.removers.remover import Remover, RemoverError
+
+
+class ListedValueRemoverError(RemoverError):
+    pass
+
+
+class ListedValueRemover(Remover):
+    def remove_listed_value(
+        self,
+        id_of_value_to_remove: str,
+    ) -> dict[str, str]:
+        removed_value_information = self._remove_from_inventory_of_listed_values(
+            id_of_value_to_remove
+        )
+        """
+        The dictionary contains three items:
+        - key: "feature_id", value: the feature ID of the removed value,
+        - key: "en", value: its English name,
+        - key: "ru", value: its Russian name.
+        """
```

Comment:
```suggestion
        removed_value = self._remove_from_inventory_of_listed_values(
            id_of_value_to_remove
        )
        """
        The dictionary contains three items:
        - key: "feature_id", value: the feature ID of the removed value,
        - key: "en", value: its English name,
        - key: "ru", value: its Russian name.
        """
```
Let's just remove `_information` from variable name then.

Also: I remembered that we now have two additional columns `description_formatted_en` and `description_formatted_ru`.

So, my suggestion is: let's make the keys in this dictionary identical to columns in `features_listed_values.csv`,
adding `id`, `description_formatted_en`, `description_formatted_ru`. And we can then write in the docstring that
the keys are identical to column names in inventory of listed values.

`id` may be excessive but very easy to fill anyway. Two other columns are actually necessary because in the future we might find ourselves removing (for future pasting) a value that has an HTML description

Reply:
Done!

Reply:
I have also thought once again of introducing `remove()` to this method (https://github.com/Jazyki-Mira/langworld_db_data/pull/301#discussion_r1841732054) but could not come up with any ways to do it. However I found a way to make the `enumerate()` cycle shorter

Reply:
If the inventory now has a slightly changed format, I think this change should also be applied to methods and tests for `ListedValueAdder`. Should I create an issue for this?

Reply:
> If the inventory now has a slightly changed format, I think this change should also be applied to methods and tests for `ListedValueAdder`. Should I create an issue for this?

Yes, please do 👍🏻